samples/dfu/src/main.c

```diff
@@ -195,6 +77,71 @@ static int data_received(struct golioth_blockwise_observe_ctx *observe,
 	return 0;
 }
 
+static uint8_t *uri_consume_trailing_slash(uint8_t *uri, size_t *uri_len)
```

Comment:
Would this be useful as a standard helper function somewhere outside the example?

Reply:
This implementation just removes first slash, so I wonder if function name describes it correctly. I am open to better name. Maybe `uri_strip_leading_slash()`?

Reply:
The alternative naming you are proposing sounds better. 

I just wonder if other examples will end up stripping the leading slash and thus increase duplicated code.

Reply:
>The alternative naming you are proposing sounds better.

Updated.

>I just wonder if other examples will end up stripping the leading slash and thus increase duplicated code.

So far we didn't need that. If we'll need that at some point, then I'll create a uri_utils.h or whatever for such things.